inc/classes/class-imagify-files-list-table.php

```diff
@@ -806,13 +806,13 @@ protected function reoptimize_buttons( $item ) {
 	}
 
 	/**
-	 * Prints a button to generate WebP versions if they are missing.
+	 * Prints a button to generate Next gen versions if they are missing.
 	 *
 	 * @since 1.7
 	 *
 	 * @param object $item The current item. It must contain at least a $process property.
 	 */
-	protected function generate_webp_versions_button( $item ) {
+	protected function generate_nextgen_versions_button( $item ) {
```

Comment:
:warning: This has been refactored here:
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-files-list-table.php#L815